### PR TITLE
docs: added an example that's compatible with cardano-addresses tool

### PR DESCRIPTION
**Pull request by AlanVerbner**
# Description

This example shows how to create and send a transaction and it is compatible with [cardano-addresses](https://github.com/input-output-hk/cardano-addresses) `Ed25519-bip32` standard. 

See https://github.com/input-output-hk/cardano-rosetta/issues/331#issuecomment-848062182 for further details.

# Proposed Solution

Emurgo serialization lib was used in order to derive and sign the transaction. It's important to mention that *key generation and the signing process is pushed to the clients by Rosetta design*. This could have been implemented using any other library or even language. `Ed25519-bip32` is not part of Rosetta scope.

# Important Changes Introduced

n/a

# Testing

Running the following `cardano-address` commands shoild result in the same addresses and keys than running this script:

```bash
echo "gasp below arrange frown canvas heart pet hole lunar card matter mom very bounce rug tobacco debris raw margin assist source also tuition cluster" > seed.txt
cardano-address key from-recovery-phrase Shelley < seed.txt > root.xsk
cardano-address key child 1852H/1815H/0H/0/0 < root.xsk | cardano-address key public --with-chain-code > addr.xvk
cardano-address address payment --network-tag testnet < addr.xvk > payment.addr
# addr_test1vqdaezgllu2g7lnkx60y7xyea4mvutxtgr49w25hrv6wfks9qn5ss
 ```

```bash

$ yarn send-transaction-using-cardano-address-example
yarn run v1.22.5
$ ts-node ./send-transaction-using-cardano-address-example.ts
[constructionDerive] Fetching an address for pub key afe915d5d360f50446e4346355f63e96537448fa6e01511a11687f30fad00c40
[constructionDerive] Retrieved address addr_test1vqdaezgllu2g7lnkx60y7xyea4mvutxtgr49w25hrv6wfks9qn5ss
[waitForBalanceToBe] Condition not met, waiting for a few seconds.
[waitForBalanceToBe] Funds found!
[waitForBalanceToBe] 1000000000 ADA
[doRun] signed transaction is 827901c03833613430303831383235383230643730616431343061346363343934313031383463336535653031613162333264623738323462393039663439356435396234343632613733343838643363663030303138313832353833393030303734336431366366653363346663633063313163323430336262633130646263376563646434343737653035333438316133363865376130366532616534346466663637373064633066346164613363663463663236303530303865323761656364623333326164333439666461373161333839666439383030323161303266616630383030333161303161353963333461313030383138323538323061666539313564356433363066353034343665343334363335356636336539363533373434386661366530313531316131313638376633306661643030633430353834303062303931393561643232373064323732636336333036633663626332353135323939643037663233343262623637313736626238313831623766653134346338633363333162643335343462623337393763363564383263626163666665633663313763353832343665343233396434333935316162656233343134613066663680
[doRun] transaction with hash 0364b82bd046a135c5e1e487367c4c863027224b5444bbcd131ae6d77c664049 sent
[waitForBalanceToBe] Condition not met, waiting for a few seconds.
[waitForBalanceToBe] Funds found!
[waitForBalanceToBe] 0 ADA
Send Transaction finished
Done in 67.92s.
```

### CODE REVIEW

**Review by james-iohk — APPROVED** [review]
Tested using my own account generated using `cardano-address`

File: examples/send-transaction-using-cardano-address-example.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+const SEND_FUNDS_ADDRESS =
+  "addr1qqr585tvlc7ylnqvz8pyqwauzrdu0mxag3m7q56grgmgu7sxu2hyfhlkwuxupa9d5085eunq2qywy7hvmvej456flknsug829n";
+
+/**
+ * The following example will work using extended keys as they were created using `cardano-address` with the following commands:
+ *
+ * echo "gasp below arrange frown canvas heart pet hole lunar card matter mom very bounce rug tobacco debris raw margin assist source also tuition cluster" > seed.txt
+ * cardano-address key from-recovery-phrase Shelley < seed.txt > root.xsk
+ * cardano-address key child 1852H/1815H/0H/0/0 < root.xsk | cardano-address key public --with-chain-code > addr.xvk
+ * cardano-address address payment --network-tag testnet < addr.xvk > payment.addr
```
Comment (james-iohk) [inline examples/send-transaction-using-cardano-address-example.ts:35]: We should move this into the example's readme.

Comment (rhyslbw) [reply examples/send-transaction-using-cardano-address-example.ts:35]: Agree

**Review by rhyslbw — CHANGES_REQUESTED** [review]
https://github.com/input-output-hk/cardano-rosetta/pull/376#discussion_r640605655

File: examples/send-transaction-using-cardano-address-example.ts (hunk 1 above)
Comment (james-iohk) [reply examples/send-transaction-using-cardano-address-example.ts]: Have added new section to readme and removed this comment block.

**Review by james-iohk — APPROVED** [review]

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**